script/openqa-bootstrap

```diff
@@ -182,3 +182,5 @@ start-worker
 if [[ $# -ne 0 ]]; then
     openqa-clone-job "$@"
 fi
+
+[[ -n $running_systemd ]] || wait
```

Comment:
we know from the current problem that `worker` will never stop by itself - wouldn't that mean `openqa-bootstrap` would again never finish but this time waiting at the end?

Reply:
Yes and that is intended, see https://progress.opensuse.org/issues/164595?#note-5

TLDR: entrypoint for openqa-single-instance container

Reply:
ah, that explains it. Would you mind adding a short comment to that line explaining it? Maybe even referencing the comment?

Reply:
awesome, thanks!